cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,76 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SettingsPageObject from '../support/pages/settings.PageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-describe('Settings page', () => {
-  before(() => {
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
 
-  });
+describe('Settings page', () => {
+  let newData;
+  let user;
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.task('generateNewData').then(generateNewData => {
+      newData = generateNewData;
+    });
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.visit();
+    settingsPage.typeNewUsername(newData.username);
+    settingsPage.clickUpdateSettingsBtn();
+    settingsPage.assertMessageAboutUpdatingData();
+    homePage.assetrNewUsername(newData.username);
   });
 
   it('should provide an ability to update bio', () => {
-
+    settingsPage.visit();
+    settingsPage.typeNewBio(newData.bio);
+    settingsPage.clickUpdateSettingsBtn();
+    settingsPage.assertMessageAboutUpdatingData();
+    homePage.clickUsernameLink();
+    settingsPage.assertMessageAboutUpdatingBio(newData.bio);
   });
 
-  it('should provide an ability to update an email', () => {
-
+  it.skip('should provide an ability to update an email', () => {
```

Comment:
The test 'should provide an ability to update an email' is skipped. If it's intentional and the test is not yet ready, remember to unskip it before the final commit.